src/components/App/index.tsx

```diff
@@ -122,13 +127,46 @@ export class AppBase extends React.Component<Props> {
     });
   }
 
+  removeError = (errorId: number) => {
```

Comment:
Nit: I don't think this function needs to be renamed from `dismissError` to `removeError`, because it's the handler for the `onClose` event and therefore the user actually is dismissing the error. I'm fine with the change, but I think `dismissError` is a better name in this instance.

Reply:
ah, I tried to come up with names that match together. Can "dismiss" be used as the opposite of "add"? (because we actually remove the error from the state).

Reply:
I marked this as a nit because it's not really that important to me. `removeError` is fine, I just think `dismissError` is better in this specific case. I still think the action should be called `removeError`, it's just the name of the function in `App` that I think could be improved. If you'd prefer to have everything match up and use `remove` that's fine with me.

Reply:
ah I see, that makes sense.